docker-compose.benchmark.yaml

```diff
@@ -387,6 +387,7 @@ services:
             - ${REDIS_PORT}
         volumes:
             - ./backend-services/redis/configs/redis.conf:/usr/local/etc/redis/redis.conf
+        command: ["redis-server", "/usr/local/etc/redis/redis.conf"]
```

Comment:
@slowr could you add this command in `redis-fix` branch to the proper k8s container place and test? It should have the same effect as in compose. Thanks!